WcaOnRails/app/assets/javascripts/registrations.js

```diff
@@ -21,7 +21,8 @@ onPage('registrations#edit_registrations', function() {
   showHideActions();
 
   $('button[value=delete-selected]').on("click", function(e) {
-    if(!confirm("Are you sure you want to delete the selected registrations?")) {
+    var $selectedRows = $registrationsTable.find("tr.selected");
+    if(!confirm("You are about to delete " + $selectedRows.length + " registrations.\nAre you sure you want to do it?")) {
```

Comment:
You have very little of the user's attention in these routine clickthrough dialogs. A lot of people will just click without reading, and I think each word adds to the percentage who do so. Anytime you can trim down the phrasing without losing information, I'll do that. Actually, come to think of it, I'd go down to this:

`"Delete the " + $selectedRows.length + " selected registrations?"`

Like you say, it's subjective, and feel free to do what you choose. This is only a suggestion from someone not doing the work.


Another thing that I would do, but you can ignore if it feels too petty is to handle the plural. That is `1 registration` vs `2 registration**s**`.